src/Components/Body/Body.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import './Body.css';
+<script src="https://cdn.jsdelivr.net/npm/kute.js@2.1.2/dist/kute.min.js"></script>
```

Comment:
**todo:** Script tags gehören in das `index.html` file. Aber in 99,9% aller Fälle kannst du stattdessen ein npm modul installieren, was deutlich sauberer ist

Reply:
Gehört zu der Animation, die ich getestet habe und sollte eigentlich commented out sein.